triage/labels.rst

```diff
@@ -10,405 +10,136 @@ Many labels are shared for both use cases, while some are dedicated
+* :gh-label:`tests`: for tests in the :cpy-file:`Lib/test` directory
+  (written in Python) and other changes related to tests, ``unittest``,
+  or ``doctests``.
 
-:gh-label:`spam`
-    Used for issues/PRs that don't include enough eggs or bacon.
 
-Labels specific to issues
-=========================
+Expert labels
+=============
+
+These labels are used to specify the area of expertise required to address
+the issue/PR.  This includes both specific modules/packages and generic
+interest areas.
 
-Priority
---------
+Adding these labels is also a way to notify the relevant experts, since
+they are encouraged to subscribe to them.  Depending on the label,
+this might also automatically add the issue to a GitHub project.
 
-:gh-label:`release-blocker`
-    The highest priority of an issue. If unaddressed, will cause the
-    release manager to hold releasing a new version of Python.
+You can see the `full list of expert labels on GitHub
+<https://github.com/python/cpython/labels?q=expert>`_.
 
-:gh-label:`deferred-blocker`
-    A release blocker that was pushed one or more releases into the
-    future. Possibly a temporary workaround was employed, or the version
-    of Python the issue is affecting is still in alpha or beta stages
-    of development.
 
-Component
----------
+OS labels
+=========
 
-:gh-label:`library`
-    Used for issues involving Python modules in the ``Lib/`` dir.
+These labels are used to specify which operating systems are affected.
+Since most issues either affect all systems or are specific to Unix,
+the only available labels are :gh-label:`OS-windows`, :gh-label:`OS-mac`,
+and :gh-label:`OS-freebsd`.
 
-:gh-label:`docs`
-    Used for issues involving documentation in the ``Doc/`` dir.
 
-:gh-label:`interpreter-core`
-    Used for issues in interpreter core (``Objects/``, ``Python/``,
-    ``Grammar/``, and ``Parser/`` dirs).
+Version labels
+==============
 
-:gh-label:`extension-modules`
-    Used for issues involving C modules in the ``Modules/`` dir.
+These labels are used to indicate which versions of Python are affected.
+The available version labels (with the form :samp:`3.{N}`) are updated
+whenever new major releases are created or retired.
 
-:gh-label:`tests`
-    Used for issues involving only Python's regression test suite, i.e.
-    files in the ``Lib/test/`` dir.
+See also :ref:`the branch status page <branchstatus>`
+for a list of active branches.
 
-Other
------
 
-:gh-label:`new`
-    Denotes that the issue hasn't been looked at by triagers or core
-    developers yet.
+.. _Keywords:
+.. _Other:
+.. _Priority:
 
-:gh-label:`easy`
-    Denotes that the issue is a good candidate for a newcomer to address.
+Other labels
+============
 
+* :gh-label:`triaged`: for issue has been accepted as valid by a triager.
+* :gh-label:`easy`: for issues that are considered easy.
+* :gh-label:`build`/:gh-label:`performance`: for issues related respectively
+  to the build process and performances.
```

Comment:
```suggestion
  to the build process or performance, respectively.
```

* Fix grammar error
* Use phrasing appropriate to the fact that these are two separate classes of labels with two separate respective applications

Reply:
There's already a "respectively" on the previous line, do you mean to remove that?

Reply:
Yes, sorry—my mistake!

The actually grammar error here wasn't actually fixed in the manual application, so I'll submit a followup suggestion and mark this one as resolved.